app/serializers/location_serializer.rb

```diff
@@ -1,10 +1,14 @@
 class LocationSerializer < ActiveModel::Serializer
-  attributes :id, :name, :location_type, :dimension, :url, :residents
+  attributes :id, :name, :type, :dimension, :url, :residents
```

Comment:
Forgot to do this in the original API code, location should return type to match the json from the original schema https://rickandmortyapi.com/documentation